src/shields-io.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const apiRoot = 'https://img.shields.io/badge/';
+const convert4shieldsIo = str => str.replace(/-/g, '--').replace(/_/g, '__');
+const toUrl = (subject, coverage, color, imageType) => `${apiRoot}${subject}-${coverage}%25-${color}.${imageType}`;
+
+module.exports = {
+  url(param) {
+    const subject = convert4shieldsIo(param.subject);
+    const coverage = convert4shieldsIo(param.coverage);
```

Comment:
`convert4shieldsIo` needs string parameter but `param.coverage` is number.
So this line rise `Type Error`, please fix this.